steps/apply/container.yaml

```diff
@@ -7,4 +7,4 @@ description: >
   This step will apply the given Terraform module or plan. The outputs of this
   step are the same as the Terraform outputs.
 settings:
-  Image: hashicorp/terraform:light
+  image: hashicorp/terraform:light
```

Comment:
This is for spindle. Since we're not using spindle anymore, you can delete this file.

Reply:
Yeah, I debated.